### DISCUSSION TITLE
Chart does not get rendered correctly

### DISCUSSION THREAD

**Discussion by matzeeable** [antvis/G2#4994] [Q&A] [answered]
Hey!

Context G2 chart: https://codesandbox.io/s/sweet-leavitt-infxuo?file=/index.ts

When I change `const days` on line `#8` to `30` I get the following chart:

![image](https://user-images.githubusercontent.com/1008534/210118027-06b6c8d6-1de3-4060-9534-a77a13bb7032.png)

But when I change it to `33`, I get the following:

![image](https://user-images.githubusercontent.com/1008534/210118053-c4871d48-9a26-470b-b0a5-7a5b0be19e64.png)

When I hover over the item, it gets rendered "slowly":

![image](https://user-images.githubusercontent.com/1008534/210118068-6c52b6a8-5dfd-44ed-b75e-33a3facc237c.png)

I already searched the issues but I could not find a similar issue. Is this a bug or do I something wrong? :-)

**Comment by hustcc** [comment, marked as answer]
```ts
.scale({
    date: {
      sync: true,
      range: [0, 1],
      type: 'cat',  // 👈🏻，here add this line.
    },
    value: {
      sync: false
    }
  });
```
